modules/langstream/pages/index.adoc

```diff
@@ -0,0 +1,312 @@
+= Getting Started with LangStream
+
+The LangStream project combines the intelligence of large language models with the agility of streaming processing, to create powerful processing applications.
+
+This guide will help you connect your LangStream application to Astra Streaming.
+
+For the LangStream project repository, see https://github.com/LangStream/langstream[GitHub^]{external-link-icon}.
+
+== Install LangStream
+
+Use the https://docs.langstream.ai/get-started[LangStream quickstart^]{external-link-icon} to install LangStream locally with Helm.
+
+== Enable LangStream in Astra
+
+xref:getting-started:index.adoc[Create an Astra Streaming tenant] in the GCP us-east-1 region. More regions will be available soon.
+
+Your tenant will be created with a default namespace, which is a logical grouping of topics. You can create additional namespaces to organize your topics.
+
+Your tenant will be listed in the LangStream tab. Select *Enable* to enable LangStream for your tenant.
+
+image::enable-langstream.png[Enable LangStream]
+
+Under the hood, this is enabling the xref:starlight-for-kafka:ROOT:index.adoc[Starlight for Kafka] API for your tenant to connect to your Kafka cluster.
+
+== Build a LangStream Application
+
+Build a LangStream application by creating YAML files to describe the application.
+The application folder structure looks like this:
+
+[source,shell]
+----
+|- sample-app
+|- application
+    |- pipeline.yaml
+    |- gateways.yaml
+|- instance.yaml
+|- configuration.yaml
+|- secrets.yaml
+----
+
+Here's a shortcut:
+[source,shell]
+----
+mkdir sample-app && cd sample-app
+touch configuration.yaml instance.yaml secrets.yaml
+mkdir application && cd application
+touch pipeline.yaml gateways.yaml
+----
+
+Next, you will populate the YAML files to connect your application to your Astra Streaming tenant.
+
+== Populate YAML files
+
+Instance.yaml declares the application's processing infrastructure, including where streaming and compute take place.
+The values for the Kafka bootstrap server are found in your Astra Streaming tenant, in the Starlight for Kafka ssl.properties file.
+The secrets for tokens and passwords are stored in the secrets.yaml file, which you'll populate in the next step.
+[source,yaml]
+----
+instance:
+  streamingCluster:
+    type: "kafka"
+    configuration:
+      admin:
+        bootstrap.servers: kafka-gcp-useast1.streaming.datastax.com:9093
```

Comment:
please update to the very latest version
https://github.com/LangStream/langstream/blob/main/examples/instances/astra.yaml

I promise it won't change anymore in the short term :-)